src/Extensions/Logging/ServiceCollectionExtensions.cs

```diff
@@ -34,14 +34,38 @@ public static ILoggingBuilder AddOmexLogging(this ILoggingBuilder builder)
 			return builder;
 		}
 
+		/// <summary>
+		/// Load the initial logger
+		/// </summary>
+		/// <param name="builder">The extension method argument</param>
+		public static ILoggingBuilder LoadInitialLogger(this ILoggingBuilder builder)
```

Comment:
nit: I would prefer `LoadInitializationLogger`, but you are better selecting appropriate naming